horovod/keras/__init__.py

```diff
@@ -108,6 +110,79 @@ def DistributedOptimizer(optimizer, name=None,
         groups=groups,
     )
 
+def PartialDistributedOptimizer(optimizer, name=None,
+                         device_dense='', device_sparse='',
+                         compression=Compression.none,
+                         sparse_as_dense=False,
+                         gradient_predivide_factor=1.0,
+                         op=Average,
+                         num_groups=0,
```

Comment:
I think we don't need to have the extra `num_groups` argument here that has been deprecated for a while in the original `DistributedOptimizer`.

(Even though nobody got around to actually removing it by 0.23)